modules/core/src/core-layers/text-layer/text-layer.js

```diff
@@ -54,23 +54,32 @@ const defaultProps = {
 
 export default class TextLayer extends CompositeLayer {
   initializeState() {
-    const {gl} = this.context;
-    const {fontFamily} = this.props;
-    const {mapping, texture} = makeFontAtlas(gl, fontFamily);
-    this.state = {
-      iconAtlas: texture,
-      iconMapping: mapping
-    };
+    this.state = {};
+    this.updateFontAtlas(this.props.fontFamily);
   }
 
   updateState({props, oldProps, changeFlags}) {
     if (
       changeFlags.dataChanged ||
+      oldProps.fontFamily !== props.fontFamily ||
```

Comment:
`transformStringToLetters()` only changes when text is changed, so `fontFamily` has no effect.

Instead, `MultiIconLayer` must invalidate `calculateInstanceOffsets` when `iconMapping` changes.